patroni/dcs.py

```diff
@@ -122,8 +122,8 @@ def __init__(self, name, config):
             i.e.: `zookeeper` for zookeeper, `etcd` for etcd, etc...
         """
         self._name = name
-        self._scope = config['scope']
-        self._base_path = '/service/' + self._scope
+        self._namespace = '/{}'.format(config.get('namespace', '/service/').strip('/'))
+        self._base_path = '/'.join([self._namespace, config['scope']])
```

Comment:
Well, the same way one could put list or dict objects there instead of str...
```yaml
scope:
  foo: bar
```

```yaml
scope: [foo, bar]
```

Such objects can be successfully converted to str, but... is this really expected behaviour?